contrib/examples/actions/workflows/customfilters/mistral-json_escape.yaml

```diff
@@ -0,0 +1,17 @@
+version: '2.0'
+
+examples.mistral-customfilters-json_escape:
+    description: Example for using the custom filter "json_escape"
+    type: direct
+    input:
+        - input_str
+    output:
+        result_jinja: <% $.result_jinja %>
+        result_yaql: <% $.result_yaql %>
+    tasks:
+
+        task2:
+            action: std.noop
+            publish:
+                result_jinja: '[{"title": "{{ _.input_str|json_escape }}"}]'
+                result_yaql: '[{"title": "<% json_escape($.input_str) %>"}]'
```

Comment:
Fixed in https://github.com/StackStorm/st2/pull/3565/commits/13a2d0d822686417bf06cffab0f10f7956366607